client/components/common/InputAsset.vue

```diff
@@ -73,7 +74,8 @@ export default {
     url: { type: String, default: null },
     publicUrl: { type: String, default: null },
     extensions: { type: Array, required: true },
-    uploadLabel: { type: String, default: 'Select file' }
+    uploadLabel: { type: String, default: 'Select file' },
+    allowFileUpload: { type: Boolean, default: true }
```

Comment:
```uploadLabel``` is used only for file upload so ```allowFileUpload``` should be placed above it (props should be ordered by importance)